### ISSUE TITLE
TypeError [ERR_INVALID_ARG_TYPE]

### ISSUE THREAD

**Issue by MXPOL** [fullstack-hy2020/bloglist-frontend#2]
I cloned the app, install its dependencies as requested but when I run the command `npm start`
I get this error

`TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined
    at validateString (internal/validators.js:117:11)
    at Object.join (path.js:1039:7)
    at noopServiceWorkerMiddleware (/home/max/Documents/js/bloglist-frontend/node_modules/react-dev-utils/noopServiceWorkerMiddleware.js:14:26)
    at Layer.handle [as handle_request] (/home/max/Documents/js/bloglist-frontend/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/max/Documents/js/bloglist-frontend/node_modules/express/lib/router/index.js:317:13)
    at /home/max/Documents/js/bloglist-frontend/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/max/Documents/js/bloglist-frontend/node_modules/express/lib/router/index.js:335:12)
    at next (/home/max/Documents/js/bloglist-frontend/node_modules/express/lib/router/index.js:275:10)
    at launchEditorMiddleware (/home/max/Documents/js/bloglist-frontend/node_modules/react-dev-utils/errorOverlayMiddleware.js:20:7)
    at Layer.handle [as handle_request] (/home/max/Documents/js/bloglist-frontend/node_modules/express/lib/router/layer.js:95:5)`

**Comment by carlank** [comment]
This is due to an issue with react-scripts version 3.3.1, where it's passing null to the middleware. Updating the dependency to 3.4.1 or greater seems to fix it. Just spent way too long trying to figure this out.

tl;dr: s/"react-scripts": "3.3.1"/"react-scripts": "3.4.1"/ in package.json
